### ISSUE TITLE
amazing paper!

### ISSUE THREAD

**Issue by SlZeroth** [TencentARC/PhotoMaker#1]
I'm waiting for release code !

I have a question about is this technology support controlnet or img2img?

thank you !

**Comment by vishaal27** [comment]
Yes, do you have an estimated timeline of release of the inference code and model checkpoints?

**Comment by zimenglan-sysu-512** [comment]
has any plan to release the training code?

**Comment by SlZeroth** [comment]
@zimenglan-sysu-512 I think code will release after 3 months

**Comment by Paper99** [comment]
Hi all,

We have released the code, demo, and model of our PhotoMaker.
